### DISCUSSION TITLE
Orange3 Software

### DISCUSSION THREAD

**Discussion by Shivam-Sharma1997** [biolab/orange3-text#1027] [General]
What is the usage of select rows and select columns in orange3 software and can i use both i the same text analysis model?

**Comment by Shivam-Sharma1997** [comment]
if orange speaks 50 language, then the dataset that includes any of those languages will be read by orange? and will that draw conclusions for us?

**Reply by PrimozGodec** [reply]
Orange can read datasets (text) in any language (also not supported). The language is important only for specific language-specific methods (keyword methods, stopword filtering, normalization, etc.). You can recognize those methods by the Language field in the widget. Methods that do not have a language field can work with any language (for example, topic modelling). What language-specific method support can be seen in the language dropdown in the widget (all the languages in the dropdown are supported by the method).

**Comment by PrimozGodec** [comment]
> What is the usage of select rows and columns in orange3 software, and can I use both in the same text analysis model?

The Select Rows widget selects only rows that match the criteria in the table. In text mining, it means that only documents that match the criteria are selected. Look for the example and documentation here: https://orangedatamining.com/widget-catalog/transform/selectrows/

With the Select Columns widget, you can remove specific columns from the table or move them to the other part of the table. For example, move the column between features to the meta part of the table. Example and documentation is here: https://orangedatamining.com/widget-catalog/transform/selectcolumns/

Sure, you can use both in the same analysis. Just connect them one after another.
